--- minigrace-js	2019-06-04 00:16:01.390392919 -0700
+++ minigrace-js-new	2019-06-04 00:17:48.414935781 -0700
@@ -5,7 +5,7 @@
     ( cd "$(dirname "$1")" || exit 1; echo "$(pwd -P)/$(basename "$1")"; )
 }
 
-HERE=`absolutePath $0`
-HERE=`dirname ${HERE}`
+HERE=`absolutePath "$0"`
+HERE=`dirname "${HERE}"`
 NODE=`which node`
-${NODE} --max-old-space-size=2048 ${HERE}/compiler-js $*
+${NODE} --max-old-space-size=2048 "${HERE}"/compiler-js $*
